scripts/install.R

```diff
@@ -0,0 +1,20 @@
+# 2016 Gregory Way
+# TAD Pathways
+# scripts/install.R
+#
+# The script installs all R dependencies required for the pipeline
+#
+# Usage:
+# Run once before implementing pipeline: "Rscript scripts/install.R"
+
+library("methods")
+
+mirror <- "http://cran.us.r-project.org"
+install.packages("checkpoint", repos = mirror)
+
+library("checkpoint")
+checkpoint("2016-02-25")
+
+library("readr")
+library("VennDiagram")
+library("dplyr")
```

Comment:
newlines at the end of script files are nice. That's why GitHub puts that ugly red symbol there.